### PR TITLE
Update the CONTRIBUTING Code Review section

### PR DESCRIPTION
**Pull request by sberyozkin**
This PR tries to add some more information to the code review process.

Guillaume, would you like to change something or may be expand even more ?

Thanks

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -153,7 +153,11 @@ We use this information to acknowledge your contributions in release announcemen
 
 ### Code reviews
 
-All submissions, including submissions by project members, need to be reviewed before being merged.
+All submissions, including submissions by project members, need to be reviewed by at least one Quarkus committer before being merged.
+
+[GitHub Pull Request Review Process](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/about-pull-request-reviews) is followed for every pull request.
+
+[Quarkus Bot](https://github.com/quarkusio/quarkus-github-bot) will make the best effort at requesting those Quarkus committers to review a given pull request who are expected to be most familiar with the affected code.
```
Comment (gsmet) [inline CONTRIBUTING.md:160]: Well, not really, PR notifications are disabled for most of the triage rules :). So I would just skip this paragraph.

**Review by gsmet — CHANGES_REQUESTED** [review]

File: CONTRIBUTING.md (hunk 1 above)
Comment (sberyozkin) [reply CONTRIBUTING.md]: @gsmet Sure, I dropped the one about `Quarkus Bot`

**Review by gsmet — APPROVED** [review]
I'll let you get it out of draft when you're ready.

**Comment by sberyozkin** [comment]
Thanks @gsmet :-), we can add more details as necessary going forward

**Pull request merged**